### DISCUSSION TITLE
Handle projects_v2_item

### DISCUSSION THREAD

**Discussion by piotrooo** [quarkiverse/quarkus-github-app#335] [Ideas]
Hello!
What do you think about add support for projectV2 events?
This will be also related with changes in github-api lib.

**Comment by gsmet** [comment]
Hello,

From what I can see, there's only one supported right now, right? https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#projects_v2_item

**Comment by piotrooo** [comment]
Yeep only this event, but really handy for automations for boards.

**Reply by gsmet** [reply]
Yeah I can certainly see that, even if I don't find the projects v2 GraphQL API very handy. I'll try to have a look next week. I'm just a bit worried that they are using the GraphQL objects so not sure they are easily mappable to what already exists in the GitHub API.

If you want to help, just gathering some payload examples with various actions would be helpful (the full payload you receive), especially to understand better what's in the `changes`.

**Reply by gsmet** [reply]
I just released Quarkus GitHub App 1.11.0 which provides support for `@ProjectsV2Item.*`. It should hit Maven Central in around 30 minutes.
